src/components/map/leaflet-areaselect.js

```diff
@@ -36,6 +38,20 @@ L.AreaSelect = L.Class.extend({
     return this;
   },
 
+  setInitialBounds: function ( bounds) {
```

Comment:
I just found my old code from another tool again:

```json
    setBounds: function(bounds) {
        if (Array.isArray(bounds) && bounds.length >= 4) {
          bounds = L.LatLngBounds([bounds[0], bounds[1]], [bounds[2], bounds[3]]);
        }
        this.map.fitBounds(bounds);

        var bottomLeft = this.map.latLngToContainerPoint(bounds.getSouthWest());
        var topRight = this.map.latLngToContainerPoint(bounds.getNorthEast());

        this.setDimensions({
            width: Math.abs(bottomLeft.x - topRight.x),
            height: Math.abs(bottomLeft.y - topRight.y)
        });
    },
```

I think this might be a bit more universal and supports the Leaflet Bounds for consistency with getBounds. Would you mind updating it?